flow/layers/platform_view_layer.cc

```diff
@@ -39,8 +39,10 @@ void PlatformViewLayer::Paint(PaintContext& context) const {
   }
   EmbedderPaintContext embedder_context =
       context.view_embedder->CompositeEmbeddedView(view_id_);
-  context.leaf_nodes_canvas = embedder_context.canvas;
-  context.leaf_nodes_builder = embedder_context.builder;
+  context.state_stack.setCanvasDelegate(embedder_context.canvas);
```

Comment:
ContainerLayer children is : [PlatformLayer, ContainerLayer]
Here we change the `context.state_stack.canvas` to `embedder.context`
In the previous version, the `internal_canvas` is a `SkNWayCanvas` which includes the frame.canvas and the embedded canvas and records the action.
I don't know if there is a potential risk here